src/xrpld/overlay/detail/ProtocolMessage.h

```diff
@@ -484,14 +468,6 @@ invokeProtocolMessage(
             success = detail::invoke<protocol::TMReplayDeltaResponse>(
                 *header, buffers, handler);
             break;
-        case protocol::mtGET_PEER_SHARD_INFO_V2:
-            success = detail::invoke<protocol::TMGetPeerShardInfoV2>(
-                *header, buffers, handler);
-            break;
-        case protocol::mtPEER_SHARD_INFO_V2:
-            success = detail::invoke<protocol::TMPeerShardInfoV2>(
-                *header, buffers, handler);
-            break;
         default:
             handler.onMessageUnknown(header->message_type);
```

Comment:
I have no strong feelings about it. I'm fine incrementing the protocol version if someone can teach me how.

Reply:
FWIW @ximinez that exactly "the peer won't be punished or booted, so not harm no foul" was my thinking when I reviewed this.

Reply:
Here's an example of how one would add a protocol version. https://github.com/ximinez/rippled/commit/f97844700229c7c0712eb79883cea2ef01b7af0f, but it's not actually useful, because older clients won't speak this protocol version, and newer clients won't support Shards anyway, so there's no need to test for it.

I updated the `PeerImp::supportsFeature` function, which for a "real" new protocol version would be called to check the feature before doing the new function or sending the new message or whatever.